openapi.yaml

```diff
@@ -205,6 +205,29 @@ components:
       description: Positive integer amount of currency. Is a string to allow bigint inputs
       format: integer
       maxLength: 64
+    Token:
+      type: object
+      required: 
+        - label
+        - address
+        - decimal
+        - icon
+      properties:
+        label: 
+          description: Token Symbol to be displayed
+          type: string
+          maxLength: 64
+        address:
+          $ref: "#/components/schemas/Address"
+        decimal: 
+          type: string
+          pattern: '/^[1-9][0-9]$'
+          description: Decimal for token
+          format: integer
+          maxLength: 64
+        icon: 
+          type: string
+          description: Token icon IPFS link
```

Comment:
Need to add a maxLength constraint here, and also make the property name a bit more descriptive like "iconHash"